src/devices/Pn532/Pn532.cs

```diff
@@ -629,7 +629,7 @@ private bool SetParameters(ParametersFlags parametersFlags)
         {
             try
             {
-                if ((toDecode[1] != 18) || (toDecode[1] != 20))
+                if ((toDecode[1] != 18) && (toDecode[1] != 20))
```

Comment:
If I'm getting this right, the method never did anything, right? The old condition is always true.

Reply:
> If I'm getting this right, the method never did anything, right? The old condition is always true.

yes